tests/fixtures/customer_cus_4UbFSo9tl62jqj.json

```diff
@@ -91,7 +91,6 @@
                 "id": "sub_fakefakefakefakefake0004",
                 "object": "subscription",
                 "application_fee_percent": null,
-                "billing": "charge_automatically",
```

Comment:
were these JSON file changes manual, or have those fields been removed from the API now? I'd rather keep the fixtures matching what the API returns so we can keep updating them automatically, even if we no longer use the fields (though I realize that updating them does require manually picking which changes are relevant, like discarding unrelated id changes, which is annoying).

Reply:
I made these changes manual. Is there any easier way to regenerate all fake fixtures? Otherwise what you say makes sense and I will revert these changes

Reply:
yep, create a throw-away Stripe account, `export DJSTRIPE_TEST_DJANGO_SECRET_KEY=...` and then  run `./manage.py regenerate_test_fixtures` - though that will result in changing some ids (since they came from my throw-away account) - probably revert those to reduce churn.

Reply:
I tried the regenerate and the field's still there, so I've updated the files back to their previous state.

So this is good to merge now.